assets/js/modules/analytics/datastore/service.js

```diff
@@ -90,7 +90,7 @@ export const selectors = {
 		let path = escapeURI`/report/${ type }/a${ accountID }w${ internalWebPropertyID }p${ profileID }/`;
 
 		if ( argsSegment ) {
-			path += escapeURI`${ argsSegment }/`;
+			path += `${ argsSegment }/`;
```

Comment:
@aaemnnosttv Does any of this actually need to be URI-escaped? Looks to me like it doesn't - maybe we should also remove that from the `path` assignment above in line 90. See @tofumatt's comment above.

Reply:
Report arg _values_ need to be URL encoded but nothing else should actually need it, assuming valid values are returned by selectors for `accountID` etc. If they weren't valid the URL wouldn't work anyways though. The `argsSegment` is already URL encoded as needed so this shouldn't have had `escapeURI` applied again because that would encode other characters in the string that shouldn't be, but it's correct now.

As far as the use in the `path` assignment, the `escapeURI` template function only encodes the variables in the string (e.g. `${ type }`), so as long as these are normal valid values it would essentially have no effect because these are all simple characters and numbers. I'm not sure it's really necessary there, but it doesn't hurt.